File: Level-0/74.java
class Solution {
    public int solution(int[] array, int n) {
        int answer = 0;
        int min = Integer.MAX_VALUE;
        for(int i = 0; i < array.length; i++){
            if(Math.abs(array[i] - n) < min){
                min = Math.abs(array[i] - n);
                answer = array[i];
            }
            if(Math.abs(array[i] - n) == min){
                answer = Math.min(answer, array[i]);
            }
        }
        return answer;
    }
}

File: Level-1/74.js
function solution(wallpaper) {
    const result = [50, 50, 0, 0];
    
    for(let i = 0; i < wallpaper.length; i++) {
        for(let j = 0; j < wallpaper[i].length; j++) {
            if(wallpaper[i][j] === '#') {
                result[0] = Math.min(result[0], i);
                result[1] = Math.min(result[1], j);
                result[2] = Math.max(result[2], i);
                result[3] = Math.max(result[3], j);
            }
        }
    }
    
    result[2] += 1;
    result[3] += 1;
    
    return result;
}
